### ISSUE TITLE
Tabs are not scrollable in desktop screens !

### ISSUE THREAD

**Issue by shivarajnaidu** [Dogfalo/materialize#4536]
### Description
I used following code to produce tabs in my web page..
When i see it in desktop screen it only showing 6 tabs instead of showing all 8 tabs with scrollbar..
But its working fine as expected in mobile..

 ```html
  <div class="row">
    <div class="col s12">
      <ul class="tabs">
        <li class="tab col s3 m2"><a href="#test1">Test 1</a></li>
        <li class="tab col s3 m2"><a class="active" href="#test2">Test 2</a></li>
        <li class="tab col s3 m2 disabled"><a href="#test3">Disabled Tab</a></li>
        <li class="tab col s3 m2"><a href="#test4">Test 4</a></li>
        <li class="tab col s3 m2"><a href="#test5">Test 5</a></li>
        <li class="tab col s3 m2"><a href="#test6">Test 6</a></li>
        <li class="tab col s3 m2"><a href="#test7">Test 6</a></li>
        <li class="tab col s3 m2"><a href="#test8">Test 7</a></li>
      </ul>
    </div>
    <div id="test1" class="col s12">Test 1</div>
    <div id="test2" class="col s12">Test 2</div>
    <div id="test3" class="col s12">Test 3</div>
    <div id="test4" class="col s12">Test 4</div>
    <div id="test5" class="col s12">Test 5</div>
    <div id="test6" class="col s12">Test 6</div>
    <div id="test7" class="col s12">Test 7</div>
    <div id="test8" class="col s12">Test 8</div>
  </div>
 ```

### Repro Steps
** Step 1 : ** `git clone https://github.com/shivarajnaidu/uv-materializecss-angularjs-template.git`
** Step 2 : ** `cd uv-materializecss-angularjs-template && npm install`
** Step 3 : ** start server by running `npm start`
** Step 4 : **   visit  http://127.0.0.1:8080 or   http://localhost:8080

### Screenshots 

![Medium Screen Only Showing 6 of 8 tabs](https://cloud.githubusercontent.com/assets/12374494/24986641/02502a98-2019-11e7-8164-f31d9367028f.png)

![Small Screen SHowing All 8 tabs With Scrollbar](https://cloud.githubusercontent.com/assets/12374494/24986682/517f7ba0-2019-11e7-9066-5e5a38875bd0.png)

**Comment by bassie1995** [comment]
Simply copying and pasting some extra tabs on the [tabs page](http://materializecss.com/tabs.html) shows that it works on the Materialize site. 
Maybe this is an issue with Angular or your use of it? Copying and pasting extra tabs in [this page](http://krescruz.github.io/angular-materialize/) doesn't work either.

**Comment by tomscholz** [comment]
@shivarajnaidu Are you using a laptop? Sometimes the scrollbar isn't displayed. Depends on the OS and browser

**Comment by shivarajnaidu** [comment]
Yes I am using laptop.. But i have tested in laptop as well as my office workstation (large screens)..
And also tested in chrome and firefox.. getting same issue...
Thanks for your response guys..

**Comment by shivarajnaidu** [comment]
Depends on the OS and browser !!!!

**Comment by brockklein** [comment]
@shivarajnaidu how did you end up enable scrolling regardless of the OS/browser?

**Comment by VeeK727** [comment]
You should remove the `col` classes from tabs. They gave me all the  trouble.
